seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/internal/dialect/oracle/OracleTypeMapper.java

```diff
@@ -0,0 +1,100 @@
+/**
+ * Copyright @ 2022科大讯飞。 All rights reserved.
+ *
+ * @author: gdliu3
+ * @Date: 2022/8/20 16:28
+ */
```

Comment:
please check this

Reply:
I'm sorry,I'll change it now

Reply:
I have doubts about the license of this code, is this a company code or yours?